backend/requirements.txt

```diff
@@ -1,32 +1,18 @@
-Flaskclick==8.1.3
```

Comment:
Please revert all the changes of this file. Main branch is already updated with a new requirements.txt